src/components/NFTCollection/types.ts

```diff
@@ -0,0 +1,71 @@
+type chain =
```

Comment:
we could add this to - so we can access it in future
https://github.com/web3ui/web3uikit/blob/master/src/web3utils/networks.ts